compiler/src/dmd/backend/cdef.d

```diff
@@ -650,19 +650,19 @@ union Vconst
         targ_llong      Vllong;
         targ_ullong     Vullong;
         Cent            Vcent;
-        targ_float      Vfloat;
-        targ_double     Vdouble;
-        targ_ldouble    Vldouble;
-        Complex_f       Vcfloat;   // 2x float
-        Complex_d       Vcdouble;  // 2x double
-        Complex_ld      Vcldouble; // 2x long double
+        targ_float      Vfloat = void;
```

Comment:
these are in a union...

Reply:
This is necessary to make the compiler believe the whole thing is zero-initialized for now. https://github.com/dlang/dmd/pull/16858 will only land in 2.111

Reply:
Oh, OK. Can you put a `FIXME` there then.

Reply:
Done